src/iiif/mailing.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+import sendgrid
+from django.conf import settings
+from sendgrid.helpers.mail import Mail
+
+log = logging.getLogger(__name__)
+
+
+def send_email(email_address, email_subject, email_body):
+    if not settings.SENDGRID_KEY:
+        log.error("No SENDGRID_KEY found. Not sending emails.")
```

Comment:
Return or raise an exception here? 

When settings.SENDGRID_KEY is None, then `sendgrid.SendGridAPIClient(settings.SENDGRID_KEY)` will look at `SENDGRID_API_KEY` inside the env. That could or could not be what you want.